src/common/OutputDataSocket.cc

```diff
@@ -299,8 +299,7 @@ int OutputDataSocket::dump_data(int fd)
 {
   m_lock.Lock(); 
   list<bufferlist> l;
-  l = data;
-  data.clear();
+  l.swap(data);
```

Comment:
@tchaikov That's a move operation. But I've been thinking, is push_back() a valid operation on a moved from object (data) ?